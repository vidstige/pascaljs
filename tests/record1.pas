programrecord1;
type
    { trailing semi}
    TPoint = record
        x, y: Integer;
    end;
    { no trailing semi}
    TName = record
        first: String;
        last: String
    end;
var
    p: TPoint;
    name: TName;
begin
    p.x := 1;
    p.y := 2;
    WriteLn('x: ', p.x, ', y: ', p.y);
    name.first := 'Peter';
    name.last := 'Forsberg';
    WriteLn(name.first, ' ', name.last);
end.