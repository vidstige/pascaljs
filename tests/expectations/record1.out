x: 1, y: 2
Peter Forsberg
